fighthealthinsurance/ml/ml_models.py

```
@@ -2328,43 +2332,109 @@ async def _resolve_tailscale_host_async(
+                logger.info(f"Resolved {hostname} to {ip_address} via Tailscale DNS")
+
+                # Cache the result
+                cls._resolved_ips[hostname] = ip_address
+                return ip_address
+        except (
+            dns.resolver.NXDOMAIN,
+            dns.resolver.NoAnswer,
+            dns.resolver.Timeout,
+            concurrent.futures.TimeoutError,
+            TimeoutError,
+            Exception,
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)